### ISSUE TITLE
v3.1.0: TypeError: Cannot read property 'replace' of undefined

### ISSUE THREAD

**Issue by XhmikosR** [MohammadYounes/rtlcss#218]
https://github.com/twbs/bootstrap/pull/32960

v3.0.0 works fine.

```css
.dropdown-menu[style] {
  right: auto/* rtl:ignore */ !important;
}
```

Do we maybe need to change the directive to be after `!important`?

/CC @ffoodd

**Comment by XhmikosR** [comment]
Actually, that must be the cause. I edited the branch and it seems to build fine.

I'm going to close the issue then.

**Comment by ffoodd** [comment]
Based on [the Changelog](https://github.com/MohammadYounes/rtlcss/blob/master/CHANGELOG.md#310---30-jan-2021), it might be a regression due to handling value directives placed after `!important`, isn't it?

@MohammadYounes your call to re-open this if you consider this a regression (since it was working fine previously).

**Comment by MohammadYounes** [comment]
@XhmikosR @ffoodd Will fix now, there shouldn't be any difference where you place the value directive.

**Comment by MohammadYounes** [comment]
@XhmikosR @ffoodd fix published in 3.1.1

Thanks.

**Comment by XhmikosR** [comment]
That was fast, thanks!
